src/main/resources/internal-admin-api.yaml

```diff
@@ -1595,6 +1597,13 @@ definitions:
         type: string
         description: The redirection URI for login
 
```

Comment:
Why Password Reset Validation, what about the password reset is being validated with this response

Can we rather just return the same object we have now, the password reset object with a redirect uri